### ISSUE TITLE
Publish2 doesn't seem to look at publish settings for collector

### ISSUE THREAD

**Issue by RicardoMusch** [diegogarciahuerta/tk-substancepainter#13]
I'm 90% sure that there is an issue with the publish2 implementation because it doesn't seem to look at the settings defined for the collector.

Whatever I do, I can't get it to look at the collector settings or the "Publish Textures as Folder" setting.

I've forked the engine and hardcoded the collector hook but it's a really weird one.

**Comment by MoisMoshev** [comment]
Publishing a textures folder seems to work out of the box. Do you mean that setting the setting to `False` does not work?
Could you be missing something in your sgtk config?

**Comment by RicardoMusch** [comment]
Yep, everything works out of the box, but any changes you make are not picked up. New plugins or settings for example are not picked up.
I had to fork the engine and hardcode whatever I needed to be changed.

**Comment by MoisMoshev** [comment]
That's odd, a cursory look at the code suggests that the settings should be used... going to have a deeper read

**Comment by RicardoMusch** [comment]
Unless I'm doing something really stupid, it's just not reading anything from settings

**Comment by RicardoMusch** [comment]
Unless I'm doing something really stupid, it's just not reading anything from settings
